src/services/completions.ts

```diff
@@ -2216,4 +2300,14 @@ namespace ts.Completions {
     function binaryExpressionMayBeOpenTag({ left }: BinaryExpression): boolean {
         return nodeIsMissing(left);
     }
+
+    function forEachAlias<T>(typeChecker: TypeChecker, symbol: Symbol, predicateMapper: (symbol: Symbol) => T | undefined): T | undefined {

```

Comment:
I would change predicateMapper to just be `predicate: (symbol: Symbol) => boolean | undefined` and then change the name to `findAlias`. At least until we need foreach+find in one function like our `forEach` function.

Reply:
Ah, yeah, I was returning something different in a previous iteration but already changed it based on some of your previous feedback. This can simplify a bit.